setup.cfg

```diff
@@ -7,6 +7,11 @@ addopts = --ignore=scripts
 [flake8]
 ignore = W503, E203, N806
```

Comment:
For clarity, I like to put each ignore code on its own line, and follow it by a comment explaining what it is and why it's being ignored. Perhaps down the line we think again about ignoring it.